sklearn/feature_extraction/image.py

```diff
@@ -499,6 +499,32 @@ def transform(self, X):
              `n_patches` is either `n_samples * max_patches` or the total
              number of patches that can be extracted.
 
+        Examples
+        --------
+
+        >>> from sklearn.feature_extraction.image import PatchExtractor
+        >>> X = np.random.randint(10, size=(2, 2, 2))
+        >>> X
+        array([[[5, 0],
+                [9, 6]],
+
```

Comment:
I don't think you need these new lines.

You can have your example in a file (e.g. `example.py`) and pass it to a script like [this](https://gist.github.com/adrinjalali/eca43556ca1620c72d91d8326ca59587) one with `python run_script.py < example.py` and copy/paste the results to the docstring.